spec/acceptance/server/grant_spec.rb

```diff
@@ -434,32 +343,6 @@ class { 'postgresql::server': }
         end
       end
 
-      it 'grant insert on single table test' do
```

Comment:
is this test checked elsewhere ?

Reply:
I merged it into the above test to grant select on a single table test.